src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/SubBsnesCore.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Nintendo.BSNES
+{
+	[PortedCore(CoreNames.SubBsnes115, "")]
+	[ServiceNotApplicable(new[] { typeof(IDriveLight) })]
+	public class SubBsnesCore
+		: IEmulator, IDebuggable, IVideoProvider, ISaveRam, IStatable, IInputPollable, IRegionable,
+			ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>, ICycleTiming
+	{
+		[CoreConstructor(VSystemID.Raw.SNES)]
+		public SubBsnesCore(CoreLoadParameters<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings> loadParameters)
+		{
+			_bsnesCore = new BsnesCore(loadParameters, true);
+			BasicServiceProvider ser = new(this);
+			ser.Register(_bsnesCore.ServiceProvider.GetService<IDebuggable>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<IVideoProvider>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<ISaveRam>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<IStatable>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<IInputPollable>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<IRegionable>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>>());
+			ser.Register(_bsnesCore.ServiceProvider.GetService<ICycleTiming>());
+			ServiceProvider = ser;
+		}
+
+		private readonly BsnesCore _bsnesCore;
+		
+		public IEmulatorServiceProvider ServiceProvider { get; }
+
+		public ControllerDefinition ControllerDefinition => _bsnesCore.ControllerDefinition;
+		public bool FrameAdvance(IController controller, bool render, bool renderSound = true)
+		{
+			return _bsnesCore.FrameAdvance(controller, render, renderSound);
```

Comment:
It would be best I think to just reimplement FrameAdvance here instead of calling the _bsnesCore one (and keeping subframe logic in here).